server/base/src/main/java/org/apache/accumulo/server/init/InitialConfiguration.java

```diff
@@ -88,9 +91,14 @@ class InitialConfiguration {
 
     initialMetaConf.put(Property.TABLE_COMPACTION_DISPATCHER.getKey(),
         SimpleCompactionDispatcher.class.getName());
-    initialMetaConf.put(Property.TABLE_COMPACTION_DISPATCHER_OPTS.getKey() + "service", "meta");
+    initialMetaConf.put(Property.TABLE_COMPACTION_DISPATCHER_OPTS.getKey() + "service",
+        DEFAULT_COMPACTION_SERVICE_NAME);
 
-    // TODO configure initial fate table config.. probably needs compaction config??
+    // ELASTICITY_TODO configure initial fate table config
```

Comment:
Is this TODO still needed? It looks like you are configuring it below

Reply:
I left the `TODO` because I wasn't sure if there were any other configuration items still needed for the fate table and didn't want it to be missed.

There seems to be a lot of settings for the root and meta tables. I wasn't sure with the introduction of the fate table if it was also getting the `initialRootMetaConf` contents set as well.

Reply:
Leaving the TODO since we are removing the compaction properties.